### PR TITLE
Make the CI great again

### PR DESCRIPTION
**Pull request by dpaneda**
Now the CI looks like a real one

### CODE REVIEW

**Comment by dagelf** [comment]
Profile pic checks out: troll

**Comment by colega** [comment]
LGTM :shipit:

File: ci/ci.sh [hunk 1]
```diff
@@ -1,3 +1,9 @@
 #!/bin/sh
 
-exit 0
+if (( RANDOM % 5 ))
```
Comment (colega) [inline ci/ci.sh:3]: Nit: it shouldn't fail on master

**Review by FriedrichVoelker — COMMENTED** [review]
LGTM

**Pull request closed without merging**